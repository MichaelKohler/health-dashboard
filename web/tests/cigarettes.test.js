/* global it, expect */
/* eslint-disable react/jsx-filename-extension */

import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Cigarettes from '../src/cigarettes.jsx';

it('renders correctly', () => {
  const tree = renderer
    .create(<MemoryRouter><Cigarettes/></MemoryRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

/* eslint-enable react/jsx-filename-extension */
