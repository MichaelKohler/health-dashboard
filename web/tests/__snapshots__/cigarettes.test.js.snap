// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section>
  <div
    className="makeStyles-appBarSpacer-1"
  />
  <div
    className="MuiContainer-root makeStyles-container-2 MuiContainer-maxWidthLg"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-md-12 MuiGrid-grid-lg-12"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation1 makeStyles-root-117 MuiPaper-rounded"
        >
          <table
            className="MuiTable-root makeStyles-table-118"
          >
            <thead
              className="MuiTableHead-root"
            >
              <tr
                className="MuiTableRow-root MuiTableRow-head"
              >
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-alignLeft MuiTableCell-sizeSmall MuiTableCell-head"
                  scope="col"
                >
                  Date
                </th>
                <th
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-alignRight MuiTableCell-sizeSmall MuiTableCell-head"
                  scope="col"
                >
                  Rolled
                </th>
              </tr>
            </thead>
            <tbody
              className="MuiTableBody-root"
            >
              <tr
                className="MuiTableRow-root"
              >
                <td
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-alignLeft MuiTableCell-sizeSmall MuiTableCell-body"
                >
                  2019-08-07T08:00:03Z
                </td>
                <td
                  aria-sort={null}
                  className="MuiTableCell-root MuiTableCell-alignRight MuiTableCell-sizeSmall MuiTableCell-body"
                >
                  not rolled
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
    <button
      aria-label="Add"
      className="MuiButtonBase-root MuiFab-root makeStyles-fab-4 MuiFab-primary"
      disabled={false}
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiFab-label"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root"
          focusable="false"
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
          />
        </svg>
      </span>
    </button>
  </div>
</section>
`;
